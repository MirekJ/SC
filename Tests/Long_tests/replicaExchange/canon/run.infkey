bf9141f7-87b9-3236-5762-b58cca2cdb94
