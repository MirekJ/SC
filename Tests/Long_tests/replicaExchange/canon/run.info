<?xml version="1.0" encoding="UTF-8"?>
<job_info>
    <infinity version="INFINITY_INFO_v_2_0" site="{ROBOX:ca923bb5-d4ed-42a3-a709-a4aae64d7931}" server="robox.ncbr.muni.cz"/>
    <basic timedate="1456145617" stimedate="2016-02-22 13:53:37">
        <external>
            <item name="INF_EXTERNAL_NAME_SUFFIX" value=""/>
            <item name="INF_EXTERNAL_VARIABLES" value=""/>
            <item name="INF_EXTERNAL_START_AFTER" value=""/>
        </external>
        <arguments>
            <item name="INF_ARG_DESTINATION" value="long"/>
            <item name="INF_ARG_JOB" value="run"/>
            <item name="INF_ARG_RESOURCES" value="ncpus=4"/>
            <item name="INF_ARG_SYNC_MODE" value=""/>
        </arguments>
        <jobinput>
            <item name="INF_JOB_PATH" value="/home/xsukenik/git/SC/SC/Tests/Long_tests/replicaExchange/canon"/>
            <item name="INF_JOB_MACHINE" value="lop.ncbr.muni.cz"/>
            <item name="INF_JOB_NAME" value="run"/>
            <item name="INF_JOB_TYPE" value="generic"/>
            <item name="INF_EXCLUDED_FILES" value=""/>
            <item name="INF_JOB_PROJECT" value=""/>
            <item name="INF_JOB_TITLE" value="run"/>
            <item name="INF_JOB_KEY" value="bf9141f7-87b9-3236-5762-b58cca2cdb94"/>
        </jobinput>
        <modules>
            <item name="INF_EXPORTED_MODULES" value=""/>
        </modules>
    </basic>
    <specific>
        <resources>
            <item name="INF_UGROUP" value="students"/>
            <item name="INF_UMASK" value="022"/>
            <item name="INF_FS_TYPE" value="consistent"/>
            <item name="INF_ALIAS" value="long"/>
            <item name="INF_ALIAS_RESOURCES" value="props=student"/>
            <item name="INF_ALIAS_SYNC_MODE" value="sync"/>
            <item name="INF_DEFAULT_RESOURCES" value="maxcpuspernode=4"/>
            <item name="INF_QUEUE" value="long"/>
            <item name="INF_NCPU" value="4"/>
            <item name="INF_NGPU" value="0"/>
            <item name="INF_MAX_CPUS_PER_NODE" value="4"/>
            <item name="INF_NNODE" value="1"/>
            <item name="INF_RESOURCES" value="nodes=1:ppn=4:student,ncpus=4,props=student,maxcpuspernode=4"/>
            <item name="INF_SURROGATE_MACHINE" value="lop.ncbr.muni.cz"/>
            <item name="INF_SYNC_MODE" value="sync"/>
            <item name="INF_SCRATCH_TYPE" value="local"/>
            <item name="INF_UGROUP_REALM" value=""/>
            <item name="INF_UGROUP_ORIG" value=""/>
        </resources>
    </specific>
    <submit timedate="1456145617" stimedate="2016-02-22 13:53:37">
        <job>
            <item name="INF_JOB_ID" value="2957.robox.ncbr.muni.cz"/>
        </job>
    </submit>
    <start timedate="1456145622" stimedate="2016-02-22 13:53:42">
        <workdir>
            <item name="INF_MAIN_NODE" value="robox2.ncbr.muni.cz"/>
            <item name="INF_WORK_DIR" value="/scratch/xsukenik/2957.robox.ncbr.muni.cz/main"/>
        </workdir>
        <nodes>
            <node name="robox2.ncbr.muni.cz"/>
            <node name="robox2.ncbr.muni.cz"/>
            <node name="robox2.ncbr.muni.cz"/>
            <node name="robox2.ncbr.muni.cz"/>
        </nodes>
        <gpus/>
    </start>
    <stop timedate="1456173613" stimedate="2016-02-22 21:40:13">
        <result>
            <item name="INF_JOB_EXIT_CODE" value="0"/>
        </result>
    </stop>
</job_info>
