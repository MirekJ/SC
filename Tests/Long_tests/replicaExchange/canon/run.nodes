robox2.ncbr.muni.cz
robox2.ncbr.muni.cz
robox2.ncbr.muni.cz
robox2.ncbr.muni.cz
