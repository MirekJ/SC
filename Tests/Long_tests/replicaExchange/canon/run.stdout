MPI SIMULATION
MPI SIMULATION
MPI SIMULATION
MPI SIMULATION

Patchy Spherocylinders version 3.6

Patchy Spherocylinders version 3.6
-------------------------------------

Patchy Spherocylinders version 3.6
-------------------------------------

Patchy Spherocylinders version 3.6
-------------------------------------
-------------------------------------
 MPI versionReading options...
 MPI versionReading topology...
Species:
 MPI versionReading topology...
Species:
 MPI versionReading topology...
Species:
 Pressure coupling type:                             1
 Pressure:                                           0.00000000
Fields:11 0
Fields:11 0
Fields:11 0
 Replica exchange pressure:                          0.00000000
Topology read of 1:        A (geotype: CPSC, 13) with parameters 1.33333 1.2 | 1.34695 0.3 | 60 5 | 0 
Topology read of 1:        A (geotype: CPSC, 13) with parameters 1.33333 1.2 | 1.34695 0.3 | 60 5 | 0 
Topology read of 1:        A (geotype: CPSC, 13) with parameters 1.33333 1.2 | 1.34695 0.3 | 60 5 | 0 


Topology read for molecule: A 

Topology read for molecule: A 
particle 1: 	particle 1: 	1 
1 
system:  A 164
Topology read for molecule: A 
particle 1: 	1 
system:  A 164
system:  A 164

Topology succesfully read. Generating pair interactions...

Topology succesfully read. Generating pair interactions...

Topology succesfully read. Generating pair interactions...
 Average volume change attempts per sweep:           0.00000000
 Equilibration sweeps:                               0
 Sweeps between step size adjustments:               0
Generating GroupLists...
Generating GroupLists...
Generating GroupLists...
 Production sweeps:                                  30000000
Probability to switch replicas is roughly: 0.985717
Probability to switch replicas is roughly: 0.985717
Probability to switch replicas is roughly: 0.985717
 Sweeps between statistics samples:                  50000
 Sweeps between statistics reports:                  50000
 Average chain move attempts per sweep:              0.00000000
 Initial maximum displacement:                       0.05000000
 Inititial maximum angular change (degrees):         7.50000000

Reading configuration...

Reading configuration...
 Inititial maximum angular cone angle (degrees):     0.00000000

Reading configuration...
 Inititial maximum geo.box edge change:              0.00000000
 Initial maximum chain displacement:                 0.00000000
 Inititial maximum chain angular change (degrees):   0.00000000
 Temperature in kT/e:                                0.24426000
 Parallel tempering temperature in kT/e:             0.28426000
 Sweeps between replica exchange:                    100000
 Sweeps between Grand-Canonical move:                0
 Sweeps between Cluster moves:                       0
 Wang-Landau method:                                 0 0
 Calculate the Wang-Landau method for atom type:     0
 Average type switch attempts per sweep:             0.00000000
 Number of Sweeps per pairlist update:               0
 Random number seed:                                 196150684
 Number of sweeps per writing out cluster info:      50000
 Sweeps between movie frames:                      50000

Reading topology...
Species:
Fields:11 0
Topology read of 1:        A (geotype: CPSC, 13) with parameters 1.33333 1.2 | 1.34695 0.3 | 60 5 | 0 

Topology read for molecule: A 
particle 1: 	1 
system:  A 164
Box: (20, 20, 20)

Topology succesfully read. Generating pair interactions...
Box: (20, 20, 20)
Box: (20, 20, 20)
Equilibration of maximum step sizes: 0 sweeps
Equilibration of maximum step sizes: 0 sweeps
Equilibration of maximum step sizes: 0 sweeps
Generating GroupLists...
Probability to switch replicas is roughly: 0.985717

Reading configuration...

Initializing energy functions...

Initializing energy functions...

Initializing energy functions...
Production run:  30000000 sweeps

Production run:  30000000 sweeps

Production run:  30000000 sweeps

Box: (20, 20, 20)
Equilibration of maximum step sizes: 0 sweeps

Initializing energy functions...
Production run:  30000000 sweeps

3 Energy drift: 6.49379e-10 
System:
A 164
3p, MPI replica changeT / changeP / acceptance ratio: 0.0133333, 0, 0.166667
2 Energy drift: 2.58933e-09 
System:
A 164
2p, MPI replica changeT / changeP / acceptance ratio: 0.0133333, 0, 0.146667
0 Energy drift: -8.74024e-10 
System:
A 164
0p, MPI replica changeT / changeP / acceptance ratio: 0.0133333, 0, 0.0466667
1 Energy drift: -5.49335e-10 
System:
A 164
1p, MPI replica changeT / changeP / acceptance ratio: 0.0133333, 0, 0.0866667

Simulation time:27960.3 s
PairList generation time:0 %

Simulation time:27959 s
PairList generation time:0 %

DONE
Deallocating Conf...

Simulation time:27962.5 s
PairList generation time:0 %

DONE
Deallocating Conf...

DONE
Deallocating Conf...

Simulation time:27962.3 s
PairList generation time:0 %

DONE
Deallocating Conf...
Deallocating Sim...
Deallocating Topo...
Deallocating Sim...
Deallocating Topo...
Deallocating Sim...
Deallocating Topo...
Deallocating Sim...
Deallocating Topo...
